.travis.yml

```diff
@@ -101,7 +101,7 @@ deploy:
     keep-history: true
     on:
       branch: release
-      condition: $DEPLOY_DOC=true
+      condition: $DEPLOY_DOC = true
```

Comment:
`on` in the `deployment` section seems to be different from the `if` elsewhere. Here is the quote form the [doc](https://docs.travis-ci.com/user/deployment/#conditional-releases-with-on):
```
condition: deploy when a single bash condition evaluates to true. This must be a string value, and is equivalent to if [[ <condition> ]]; then <deploy>; fi. For example, $CC = gcc.
```

Here is a working build: https://travis-ci.org/RustPython/RustPython/builds/478843944 . The deployment steps failed because I set a bad token to avoid actually deploying. But the deployment step seems to kick-off correctly.

Reply:
Wow, this is pretty crappy, but good research!